tests/test_schema.py

```diff
@@ -1462,6 +1463,14 @@ async def check_http_headers(c):
     assert res.status == 200
     assert res.headers["Content-Type"] == "application/vnd.schemaregistry.v1+json"
     assert res.json()["compatibility"] == "NONE"
+    # Subjects request with no headers will return a valid response
+    # using requests directly as aiohttp test client seems to inject a header
+    # thus you can only test this remotely
+    base_url = os.getenv("SERVER_URI", "")
+    if baseurl:
+        path = os.path.join(base_url, "subjects/unknown_subject")
+        res = requests.request("POST", path, data=b"{}", headers={})
```

Comment:
Hmm this is without any content-type, could you also add a something like cOnTeNt-TyPe: as one to make sure we have an explicit test for the casing too.